docs/glossary.md

```diff
@@ -0,0 +1,41 @@
+<a id="top"></a>
+
+# Glossary of Terms
+
+
+| MLRun Terms              | Definition                                                                                                                                                                                    |
+| -------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| FeatureSet               | Group of features that are ingested together and stored in logical group                                                                                                                      |
+| FeatureVector            | Combination of multiple FeatureSets with ability to join, filter, etc.                                                                                                                        |
+| MLRun Function           | Abstraction that allows for running Python code easily in local environment and on top of K8s                                                                                                 |
```

Comment:
its not Py specific, use "Abstraction over the code, extra packages, runtime configuration and desired resources which allow execution in local environment and on various serverless engines on top of K8s"